streams/src/test/java/org/apache/kafka/streams/tests/StreamsBrokerDownResilienceTest.java

```diff
@@ -30,10 +30,7 @@
 import org.apache.kafka.streams.kstream.ForeachAction;
 
 import java.io.IOException;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Properties;
+import java.util.*;
```

Comment:
Build fails because of this -- don't use wildcard imports.

Reply:
@mjsax understood; will update my IDE to avoid doing that.